11-parameter-store/README.md

```diff
@@ -197,11 +199,15 @@ Read [Using Dynamic References to Specify Template Values](https://docs.aws.amaz
 Why can't you use that feature directly to read "middle-name" and show it
 in your web page?
 
+> It looks like we can but it is can be dangerous as it may leak secret keys
+
 #### Question 2
 
 Can you use Secure String as an `AWS::SSM::Parameter::Value` type in a
 CloudFormation stack?
 
+> No
+
```

Comment:
The [AWS::SSM::Parameter ](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ssm-parameter.html)docs say, "AWS CloudFormation doesn't support the SecureString parameter type", but that’s obsolete as of [Aug 2018](https://aws.amazon.com/about-aws/whats-new/2018/08/aws-cloudformation-introduces-dynamic-references-to-support-aws-/). According to the CloudFormation [Parameters](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html#aws-ssm-parameter-types) doc:

> AWS CloudFormation does not support defining template parameters as SecureString Systems Manager parameter types. However, you can specify Secure Strings as parameter values for certain resources by using dynamic parameter patterns. For more information, see Using Dynamic References to Specify Template Values.